packages/vue/src/auth/mapAuthState.ts

```diff
@@ -127,6 +131,24 @@ export const useFronteggLoaded = () => {
   return fronteggLoaded;
 };
 
+/**
+  @param keys The requested entitlement keys
+  @returns Entitlements contain state data for every key (inc if entitled)
+*/
+export const useEntitlements = (keys: string[]): Entitlements => {
+  const authState = inject(authStateKey);
```

Comment:
I would create `entitlementsStateKey` and then use it here
off course you have to provide it before like so - 
`      Vue.provide(entitlementsStateKey, authState.entitlementsState?.entitlements || {});
`